pallets/grants/src/lib.rs

```diff
@@ -261,6 +259,10 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn renounced)]
+	pub type Renounced<T: Config> = CountedStorageMap<_, Blake2_128Concat, T::AccountId, bool, ValueQuery>;
```

Comment:
Just wondering why we'd need a `CountedStorageMap` here? Could a`StorageMap` work as well, considering it's a bit lighter too.

Reply:
Honestly copy and pasted this from the vesting schedule storage map. Hadn't realized it wasn't a pure StorageMap. Will fix